pom.xml

```diff
@@ -31,6 +31,7 @@
     <maven.compiler.release>17</maven.compiler.release>
 
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.build.outputTimestamp>2024-02-11T10:52:01Z</project.build.outputTimestamp>
```

Comment:
Hi @hboutemy , thanks for the PR.
Since we use git commit-id plugin I think it is more relevant to use last git commit time which will enable to get a dynamic but reproducible build date rather than something we'll likely not maintain manually and which can break the build in time (having a moving build time is key for artifacts). 
wdyt?